optaplanner-wb-webapp/pom.xml

```diff
@@ -546,6 +546,26 @@
       <artifactId>kie-wb-common-library-backend</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-client</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.screens</groupId>
+      <artifactId>kie-wb-common-examples-screen-backend</artifactId>
+    </dependency>
+
+    <!-- KIE Workbench Playground Repository -->
+    <dependency>
+      <groupId>org.kie.workbench.playground</groupId>
+      <artifactId>playground-parent</artifactId>
```

Comment:
The playground-parent's pom.xml's "packaging" type is "pom", yet that pom.xml has a side effect of building and deploying a jar too. Combine that with the fact that "packaging" defaults to "jar" and this works.

So maybe we should split up playground-parent into kie-playground-parent and kie-playground-distribution? The modules code would go into parent, but the assembly and xslt code would go into playground-distribution. optaplanner-wb would depend on distribution instead of parent.